Operating Systems/concepts.txt

```diff
@@ -81,11 +81,11 @@ It is very much needed to seprate these two inorder to reach maximum flexibility
 
 OS STRUCTURE: 5 basic structures: 
 
-MONOLYTHIC: Doesnot break into subsystems and has no distinction between user and kernel mode. Provides direct access to hardware.
+MONOLITHIC: Doesnot break into subsystems and has no distinction between user and kernel mode. Provides direct access to hardware.
 
-LAYERED: Another approach is to create multiple small layers. In which each layer depends upon the services the layer below. No lower layer can use services of upper layer. This can be inefficient as request for service from higher layer has to filter all lower layers.
+LAYERED: Another approach is to create multiple small layers. In which each layer depends upon the services the layer below. No lower layer can use services of upper layer. This can be inefficient as request for service from higher layer has to filter all lower layers but it does provides abstraction makes the code less cohesive.
```

Comment:
`but it does provides abstraction makes the code less cohesive.` -> `but it does provide abstraction whihc makes the code less cohesive`.